server/modules/user/api/userApi.js

```diff
@@ -97,7 +112,10 @@ module.exports.init = app => {
 
       const { user } = req
       const { surveyId, userUuid, name, email, groupUuid } = Request.getParams(req)
-      const updatedUser = await UserService.updateUser(user, surveyId, userUuid, name, email, groupUuid)
+
+      const fileReq = (req.files && req.files.picture) || null
```

Comment:
don't we have Request.getFile?

Reply:
Yes, using getFile and renamed image http param to file